pkg/cli/image/mirror/mirror.go

```diff
@@ -735,6 +735,9 @@ func copyManifestToTags(
 		for _, desc := range srcManifest.References() {
 			plan.parent.parent.AssociateBlob(desc.Digest, plan.parent.name)
 		}
+		if srcDigest != toDigest {
+			fmt.Fprintf(out, "warning: Digests are not preserved with schema version 1 images. Support for schema version 1 images will be removed in a future release.\n")
```

Comment:
Wire `errout` and use that, from what I'm grepping through our commands we tend to warn through stderr, not stdout.

Reply:
fixed, done